architecture/backend_upstream.md

```diff
@@ -0,0 +1,24 @@
+# Backend and Upstream Testing
+
+Sometimes you need to debug an issue on production without varnish being in your way.
```

Comment:
maybe explain shortly that production have varnish enabled and development (and also local development environemnts) have varnish around but disabled